webapp/src/components/jira_field.jsx

```diff
@@ -89,7 +98,10 @@ export default class JiraField extends React.PureComponent {
 
         // if this.props.field has allowedValues, then props.value will be an object
         if (field.allowedValues && field.allowedValues.length && field.schema.type !== 'array') {
-            const options = field.allowedValues.map(this.makeReactSelectValue);
+            let options = field.allowedValues.map(this.makeReactSelectValue);
+            if (field.name === 'Priority') {
```

Comment:
Why are we special casing Priority? 
Why can't we follow the example here? https://github.com/JedWatson/react-select/blob/master/docs/examples/CustomOption.js

Reply:
@crspeller FYI from the commit message (if relevant)

> Note : priority is the only dropdown field that does not currently support searching by text. This is because options.label expects a string, not a fragment.

Reply:
I'm not seeing how to remove the `priority` special casing, even with that example.  (I want to, but don't see it).